### PR TITLE
tickets/SITCOM-275

### PR DESCRIPTION
**Pull request by mareuter**
Remove extra ack in base_make_calibrations.py.

### CODE REVIEW

File: python/lsst/ts/externalscripts/base_make_calibrations.py [hunk 1]
```diff
@@ -493,11 +494,6 @@ async def call_pipetask(self, image_type, exposure_ids_dict):
             data_query=f"instrument='{self.instrument_name}' AND"
             f" detector IN {self.config.detectors} AND exposure IN {exposure_ids}",
         )
-        if ack.ack != salobj.SalRetCode.CMD_ACK:
-            ack.print_vars()
-
-        # Wait for the in-progress acknowledgement with the job identifier.
-        ack = await self.ocps.cmd_execute.next_ackcmd(ack, wait_done=False)
         self.log.debug(
```
Comment (tribeiro) [inline python/lsst/ts/externalscripts/base_make_calibrations.py:497]: I think we should probably remove the following debug statement...

File: python/lsst/ts/externalscripts/base_make_calibrations.py [hunk 2]
```diff
@@ -619,11 +615,6 @@ async def verify_calib(self, image_type, job_id_calib, exposure_ids_dict):
             data_query=f"instrument='{self.instrument_name}' AND"
             f" detector IN {self.config.detectors} AND exposure IN {exposure_ids}",
         )
-
-        if ack.ack != salobj.SalRetCode.CMD_ACK:
-            ack.print_vars()
-
-        ack = await self.ocps.cmd_execute.next_ackcmd(ack, wait_done=False)
         self.log.debug(
```
Comment (tribeiro) [inline python/lsst/ts/externalscripts/base_make_calibrations.py:618]: Another debug to be removed?

**Review by tribeiro — APPROVED** [review]
Looks good, I made a few comments about some debug messages left over. I think they should be removed.

File: python/lsst/ts/externalscripts/base_make_calibrations.py (hunk 1 above)
Comment (mareuter) [reply python/lsst/ts/externalscripts/base_make_calibrations.py:497]: I'm not sure about removing them. They were after the original in-progress ack retrieval so I think leaving them there in the new context still allows one to diagnose a problem if something goes wrong with the command exec. That's how I was able to determine what broke when I went to fix this.

**Pull request merged**